bundle/edu.gemini.ags.client.impl/src/main/scala/edu/gemini/ags/client/impl/AgsHttpClient.scala

```diff
@@ -19,9 +24,12 @@ object AgsHttpClient {
   private val hostnameVerifier: HostnameVerifier = new HostnameVerifier {
      def verify(s: String, sslSession: SSLSession) = true
   }
+
+  // LRU cache to minimize lookups.
+  val lRUCache = Collections.synchronizedMap(new LruCache[String, AgsResult.Success](100))
```

Comment:
Unsure about size here. I figured 100 seemed more than large enough, and given the size of the objects being stored in the cache (query `String's as keys and wrapped `Double` guiding estimates as values), this was not problematically large.